components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/api/WorkersApiService.java

```diff
@@ -29,7 +30,7 @@
 
     public abstract Response deleteWorker(String id) throws NotFoundException, SQLException;
 
-    public abstract Response enableSiddhiAppStats(String id,String appName,boolean statEnable)
+    public abstract Response enableSiddhiAppStats(String id,String appName,StatsEnable statEnable)
```

Comment:
format this.. space after the comma, check the whole PR

Reply:
Fixed in ea04a21de7eed189e8627127fcd8e4c9c600b999